### DISCUSSION TITLE
Automatically choose default profile?

### DISCUSSION THREAD

**Discussion by richiema32** [UltimaHoarder/UltimaScraper#1110] [Ideas]
Read through the FAQ, didn't see if it was already a thing but similar to how we can set for onlyfans to be the automatic website, can there be an option for which account is chosen automatically? 

Basically want to dumb down the process to just start it up and it authorises lol.

**Comment by M-rcus** [comment]
In `.settings/config.json` there's an option called `auto_profile_choice`, set it to `"default"` (or the name of your profile - if you have a different name).

Snipped example below, so you can see where the setting is located:
```json
{
    "supported": {
        "onlyfans": {
            "settings": {
                "auto_profile_choice": "default"
            }
        }
    }
}
```
